parent/pom.xml

```diff
@@ -1072,6 +1073,12 @@
         <artifactId>swagger-annotations</artifactId>
         <version>${version.swagger-annotations}</version>
       </dependency>
+
+      <dependency>
+        <groupId>org.checkerframework</groupId>
+        <artifactId>checker-qual</artifactId>
+        <version>${version.checker-qual}</version>
+      </dependency>
```

Comment:
:information_source: It looks like that the checker-framework is released under several different licenses one is GPL2
https://github.com/typetools/checker-framework/blob/master/LICENSE.txt

The dependency we use or defined here is licensed under MIT https://github.com/typetools/checker-framework/blob/master/checker-qual/LICENSE.txt just for posterity

@oleschoenburg mention this was already part of our transitive dependencies, but in order to resolve conflicts we had to define it here.